pkg/uefi/biosregion.go

```diff
@@ -56,15 +57,35 @@ type BIOSRegion struct {
 	//Metadata for extraction and recovery
 	ExtractPath string
 	Length      uint64
-	// This is a pointer to the Region struct laid out in the ifd
-	Position *Region `json:",omitempty"`
+	// This is a pointer to the FlashRegion struct laid out in the ifd
+	flashRegion *FlashRegion
+	RegionType  FlashRegionType
 }
 
-// NewBIOSRegion parses a sequence of bytes and returns a BIOSRegion
+// Type returns the flash region type
```

Comment:
Put a period at the end of this sentence.

Reply:
Done